pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1955,6 +1957,13 @@
             "org.apache.pulsar.broker.service.schema.ProtobufNativeSchemaCompatibilityCheck"
     );
 
+    @FieldContext(
+            category = CATEGORY_SCHEMA,
+            doc = "The schema compatibility strategy in broker level. If this config in namespace policy is `UNDEFINED`"
+                    + ", schema compatibility strategy check will use it in broker level."
+    )
+    private String schemaCompatibilityStrategy = "UNDEFINED";
```

Comment:
Could we use the name of the enum `SchemaCompatibilityStrategy.UNDEFINED`.

Reply:
good idea

Reply:
We should be able to directly  use the enum type here